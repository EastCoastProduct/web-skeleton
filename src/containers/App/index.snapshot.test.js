import React from 'react';
import renderer from 'react-test-renderer';
import { AppComponent } from './';

describe('App component snapshot', () => {
  it('renders', () => {
    const children = <div>Test</div>;
    const tree = renderer.create(
      <AppComponent
        children={children}
        dispatch={() => {}}
        router={{}}
      />
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
