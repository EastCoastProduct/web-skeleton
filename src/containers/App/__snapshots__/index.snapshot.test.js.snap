exports[`App component snapshot renders 1`] = `
<div>
  <nav>
    <a
      onClick={[Function]}
      style={Object {}}>
      Profile
    </a>
    <a
      href={true}
      onClick={[Function]}>
      Logout
    </a>
  </nav>
  <main>
    <div>
      Test
    </div>
  </main>
</div>
`;
